src/Pages/Main/Components/Nav/Nav.scss

```
@@ -0,0 +1,239 @@
+@import '../../../../Styles/variables.scss';
+
+a {
+  text-decoration: none;
+  color: $black;
+}
+
+.Nav {
+  .mainContainer {
+    display: flex;
+    justify-content: center;
+    height: 75px;
+    border-bottom: 1px solid $lightGrey;
+  }
+
+  .subContainer {
+    display: flex;
+    align-items: center;
+  }
+
+  // left
+  .navBarLeftContainer {
+    display: flex;
+  }
+
+  .navBarLeft {
+    display: flex;
+    align-items: center;
+
+    li {
+      margin: 0px 15px 0px 15px;
+    }
+  }
+
+  .navBarHover:hover {
+    padding: 28px 0px 28px 0px;
+    border-bottom: 2px solid $black;
+    .navDropdown {
+      display: block;
+    }
+  }
+
+  .dropdownContainer {
+    display: flex;
+    justify-content: space-evenly;
+    position: absolute;
+    padding-top: 50px;
+    margin-top: 30px;
+    left: 0px;
+    width: 100%;
+    height: 500px;
+    border: black solid;
+    background-color: $lightGrey;
+  }
+
+  .navDropdown {
+    // display: none;
+    display: block;
+    z-index: 1;
+  }
+
+  .allItems {
+    .allItemsTitle {
+      font-weight: bold;
+    }
+  }
+
+  .elecItems {
+    .elecItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
+      text-align: left;
+    }
+  }
+
+  .wearItems {
+    .wearItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
+      text-align: left;
+    }
+  }
+
+  .livingItems {
+    .livingItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
+    }
+  }
+
+  .dropdownCharacters {
+    display: flex;
+
+    .dropdownCharactersTitle {
+      position: absolute;
+      top: 300px;
+      font-weight: bold;
+      width: 100px;
+    }
+
+    ul {
+      display: table;
+    }
+
+    li {
+      display: inline;
+      top: 500px;
+    }
+  }
+
+  .navBarBest:hover {
+    padding: 28px 0px 28px 0px;
+    border-bottom: 2px solid $black;
+  }
+
+  // header
+  .navBarHeader {
+    padding: 0px 250px 0px 350px;
+
+    a {
+      font-weight: bold;
+    }
+  }
+
+  // right
+  .navBarRightContainer {
+    display: flex;
+  }
+
+  .navBarRight {
+    display: flex;
+    align-items: center;
+
+    li {
+      margin: 0px 15px 0px 15px;
+    }
+  }
+
+  .navBarSearch {
+    position: relative;
+  }
+
+  .searchContainer {
+    input {
+      margin: 0px 10px 0px 10px;
+      outline: 0px;
+      border: none;
+      border-bottom: $lightGrey solid 1px;
+    }
+  }
+
+  .searchAndRecentContainer {
+    background-color: $black;
+  }
+
+  .searchContainer:hover {
+    .searchTrend {
+      display: block;
+    }
```

```suggestion
  .searchContainer {
    input {
      margin: 0px 10px 0px 10px;
      outline: 0px;
      border: none;
      border-bottom: $lightGrey solid 1px;
    }
    
    &:hover {
      .searchTrend {
        display: block;
      }
    }

  .searchAndRecentContainer {
    background-color: $black;
  }
```
- & 연산자 사용해서 nesting 해줄 수 있습니다
- 같은 `.searchContainer`인데 서로 떨어져 있네요! 유사한 내용은 같은 자리에 있어야 내용을 읽거나 유지보수하기에 좋을 것 같습니다 :)

---

& 연산자 사용해서 nesting 해주겠습니다!